test/presenters/supergroups/policy_and_engagement_test.rb

```diff
@@ -12,21 +12,177 @@
-            path: '/government/tagged/content'
-          },
-          metadata: {
-            public_updated_at: '2018-02-28T08:01:00.000+00:00',
-            organisations: 'Tagged Content Organisation',
-            document_type: 'Case study'
+      assert_equal expected_result('case_study'), policy_and_engagement_supergroup.document_list(taxon_id)
+    end
+
+    describe 'consultations' do
+      before do
+        content = content_item_for_base_path('/government/tagged/content').merge(
+          "details": {
+            "body": "",
+            "closing_date": "2017-07-10T23:45:00.000+00:00"
           }
-        }
-      ]
+        )
+
+        content_store_has_item('/government/tagged/content', content)
+      end
+
+      it 'prioritises consultations over other content' do
+        tagged_document_list = %w(
+          consultation_outcome
+          case_study
+          open_consultation
+          case_study
+          closed_consultation
+        )
+
+        expected_order = %w(
+          consultation_outcome
+          open_consultation
+          closed_consultation
+          case_study
+          case_study
+        )
+
+        MostRecentContent.any_instance
+          .stubs(:fetch)
+          .returns(tagged_content(tagged_document_list))
+
+        assert_equal expected_results(expected_order), policy_and_engagement_supergroup.document_list(taxon_id)
+      end
+
+      describe '#special_format_count' do
+        it 'only show applies special formatting to consultations' do
+          tagged_document_list = %w(
+            case_study
+            case_study
+            case_study
+            consultation_outcome
+            closed_consultation
+          )
+
+          MostRecentContent.any_instance
+            .stubs(:fetch)
+            .returns(tagged_content(tagged_document_list))
+
+          assert_equal 2, policy_and_engagement_supergroup.special_format_count(taxon_id)
+        end
+
+        it 'only show applies special formatting to the first three consultations' do
+          tagged_document_list = %w(
+            consultation_outcome
+            closed_consultation
+            open_consultation
+            consultation_outcome
+            closed_consultation
+          )
+
+          MostRecentContent.any_instance
+            .stubs(:fetch)
+            .returns(tagged_content(tagged_document_list))
+
+          assert_equal 3, policy_and_engagement_supergroup.special_format_count(taxon_id)
+        end
+      end
+
+      describe '#consultation_closing_date' do
+        it 'gets the closing date of past consultations' do
+          MostRecentContent.any_instance
+            .stubs(:fetch)
+            .returns(section_tagged_content_list('open_consultation'))
+
+          assert_equal expected_result('open_consultation'), policy_and_engagement_supergroup.document_list(taxon_id)
+        end
+
+        it 'gets the closing date of future consultations' do
```

Comment:
I'm a bit confused about the difference between this test and `gets the closing date of past consultations`. We seem to be testing both with a document type of `open_consultation`, but the test names suggest we're testing past (closed consultations) and future (open consultations)?

Reply:
@vanitabarrett The naming is bad. The description of the consultation closing date changes depending on whether it's in the past or future.

e.g. "Closing date 10 July 2018" vs "Date closed 10 July 2017"

In `gets the closing date of past consultations` the closing date is in 2017, and in `gets the closing date of future consultations` the date is in 2018.